src/dss/manifest_templates/dss_core.yaml.j2

```diff
@@ -5,3 +5,17 @@ metadata:
   name: {{ namespace }}
   labels:
     app.kubernetes.io/part-of: dss
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ notebook_pvc_name }}
+  namespace: {{ namespace }}
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
This worries me a bit. For now because of microK8s architecture it won't matter because the PVC will grow outside this limit either way.

I'd suggest that we move this to 10Gi early on, so even when we have to make the update in the future it won't bite us that much that users will only be getting 1Gi by default if the underlying logic of microK8s changes

Reply:
Happy to change it, but I don't think this has negative consequences as is.

It is at 1Gi now arbitrarily because, like you said, for out given storage class it has no effect.  When we move to a different storage class I think things will have to change regardless (ex: `storageClassName: microk8s-hostpath` will change, and we probably need user input about storage size rather than just a fixed amount).  So to me, whatever we choose here has no effect on anything :D  

But for that reason, I've changed it haha